src/jvm/main/org/jetbrains/kotlinx/lincheck/LinChecker.kt

```diff
@@ -100,18 +107,20 @@ class LinChecker(private val testClass: Class<*>, options: Options<*, *>?) {
             if ((i + 1) % VERIFIER_REFRESH_CYCLE == 0)
                 verifier = createVerifier()
             scenario.validate()
-            reporter.logIteration(i + 1, scenariosSize, scenario)
-            var failure = scenario.run(this, verifier)
+            reporter.logIteration(i, scenariosSize, scenario)
```

Comment:
No it is not a but, I've just moved `+ 1` into `logIteration` itself. 
The reason is that `i` is the `0`-starting iteration number. 
The fact that we print e.g. `Iteration 1` instead of `Iteration 0` in the logs is just an implementation detail of the reporter itself.